gin-sample/main.go

```diff
@@ -51,7 +52,7 @@ func main() {
 	router.GET("/sign-in", func(ctx *gin.Context) {
 		session := sessions.Default(ctx)
 		logtoClient := client.NewLogtoClient(logtoConfig, &SessionStorage{session: session})
-		signInUri, err := logtoClient.SignIn("http://localhost:8080/sign-in-callback")
+		signInUri, err := logtoClient.SignIn(os.Getenv("CALLBACK_URI"))
```

Comment:
```suggestion
		signInUri, err := logtoClient.SignIn(os.Getenv("REDIRECT_URI"))
```
To keep align with the field in the console:
<img width="772" alt="image" src="https://github.com/logto-io/go/assets/10806653/ba9fd4b9-976c-4d0a-bf3d-c1269418e861">

Reply:
Done.